src/GNUmakefile

```diff
@@ -15,11 +15,14 @@ OBJCOPY  = $(CROSS_COMPILE)objcopy
 OBJDUMP  = $(CROSS_COMPILE)objdump
 
 HAS_SWIG := $(shell swig -version 2>/dev/null)
-HAS_PYTHON_CONFIG := $(shell python2.7-config --ldflags 2>/dev/null)
+PYTHON_VERSION=$(shell python3 --version | cut -f 2 -d ' ' | cut -f 1,2 -d'.')
```

Comment:
Can we use `python3-config` here instead of `cut`? Also, why don't we set `PYTHON="python3"`, then call `"${PYTHON}"-config` in `PYTHON_VERSION`.

Reply:
Partly my bad here, I wanted this to be `python --version` and it would pick up whatever was setup in PATH. Newer systems have `/usr/bin/python` pointing to `/usr/bin/python3`.

I hadn't noticed `python3-config` on my machine, but looks like that does point to `python${PYTHON_VERSION}-config`, so I'll make the changes